app/konnect/analytics/custom-reports/use-cases.md

```diff
@@ -0,0 +1,223 @@
+* **Then by**: None
+
+[screenshot of finished report]
+
+This report can provide your stakeholders with the answers to questions like:
+
+* How many requests were made to the API during the current month? 
+* How does this month compare to the previous months? 
+* Which service receives the most requests?
+
+The data can be used to support other types of exploratory questions about your API: 
+
+* What is the most popular service? 
+* Are there spikes in requests across months? 
+* Are there any errors or issues that might be impacting the number of requests our API is receiving? 
+
+## Daily API usage for an API over the last 30 days
+
+From the previous report, you determine that the Accounts API is receiving the most traffic. 
+You don't know whether this is a cause for concern or not, so you decide to take a closer look.
+
+To gain further insight into the Accounts API, let's build a report that displays the number of requests per day for the Accounts API over the last 30 days. Set the following options in the UI:
+
+* **Name**: Daily Accounts API Usage (last 30 days)
+* **Chart type**: Line chart
+* **Date/Time**: Last 30 days
+* **Metric**: Request count
+* **Group by**: Service
+
+Add a filter for the Account API. Click on **Add Filter**, then set the following options:
+
+* **Filter by**: Service
+* **Filter value**: Accounts API (or your own API name)
+
+[screenshot of finished report]
+
+This report can provide your stakeholders with the answers to questions like:
+
+* What does average traffic over time look like for this service?
+* Are there any anomalies, such as spikes or drops in traffic?
+* If there are anomalies, do they repeat? Are there any patterns in the data?
+* If there's an issue with API traffic, has it been resolved or is it ongoing?
+
+## Daily API usage by application over the last 30 days
+
+At this point you have two custom reports: 
+
+* Total number of requests across APIs. 
+* Daily API Usage for an API over the last 30 days.
+
+These two custom reports can work together to provide you with a lot of information about your organizations product. These alone can help drive a lot internal change to your API. Let's dive deeper, if you want to learn about the types of third-party applications that your users use to connect to your API, set the following options in the UI: 
+
+* **Name**: API Usage by Application (last 30 days)
+* **Chart type**: Vertical bar chart
+* **Date/Time**: Last 30 days
+* **Metric**: Request count
+* **Group by**: Service
+* **Then by**: Application
+
+[screenshot of finished report]
+
+This report can be used to highlight which applications users prefer, where to distribute resources, and combined with the other reports, you can now communicate usage data about your API to your stakeholders. 
+## Latency for an API over the last 30 days
+
+This use case demonstrates how you can use custom reports to track the latency for an API, the Payment API in this case, over the last 30 days.
```

Comment:
@Guaris @lena-larionova I'm not sure if I should keep these as steps or not. I like the format of steps, but I don't think a user could follow this from start to finish because I never explain how to set up the "Payment" API. I feel like that's a bit outside of the scope of this section too. 

I think I could either add in info about how to create the Payment API and leave the steps, or match the sections you wrote where we don't have steps and it's more casual of "here's how you'd configure this" vs. "follow this from step 1 to step 2". Thoughts?

Reply:
> or match the sections you wrote where we don't have steps and it's more casual of "here's how you'd configure this" vs. "follow this from step 1 to step 2". Thoughts?

I think this is the way to go. I don't think creating/configuring a service is in scope here. 

What we can do is introduce the APIs in the narrative. You're this person, and you have these APIs that you're responsible for: Payments, Accounts, and <something>.

Reply:
I agree with Lena.